apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestDataPathsActivity.java

```diff
@@ -112,6 +110,14 @@ public static String comparePassedField(String prefix, Object failed, Object pas
         }
     }
 
+    public static double calculatePhaseError(double p1, double p2) {
```

Comment:
The diff would be negative if the diff is over 2PI. Should we do some math wrapping around if it exceeds 2PI?

Reply:
Thanks. The atan2() function will return angles in a reasonable range. But this function does not know where the angles came from and should be more robust.